### ISSUE TITLE
Interpreter Console only accept basic expressions

### ISSUE THREAD

**Issue by joey-coleman** [symphonytool/symphony#171]
> The following bug was [originally reported](https://sourceforge.net/p/compassresearch/bugs/171/) on Sourceforge by [anderskm](https://sourceforge.net/u/anderskm/profile/), 2013-11-27 13:45:52.405000:

The CmlInterpreter only accept basic values and expressions (with a few exceptions where CML and VDM syntax differs) when parsing input from the user. Values of named types is not working

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/171/#90de) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-29 00:40:14.625000:
- **summary**: The CmlInterpreter only accept basic values and expressions (with a few exceptions where CML and VDM syntax differs) when parsing input from the user. Values of named types is not working --> Interpreter Console only accept basic expressions
- Description has changed:

Diff:

```

--- old
+++ new
@@ -0,0 +1 @@
+The CmlInterpreter only accept basic values and expressions (with a few exceptions where CML and VDM syntax differs) when parsing input from the user. Values of named types is not working

```
- **status**: open --> accepted
- **assigned_to**: Kenneth Lausdahl

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/171/#2611) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-11-29 00:40:15.671000:

Updated the parser to CML parser [9c7a4b182a0723b5b34d77948afce8262ff463c3]

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/171/#10f1) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-12-04 09:37:45.756000:
- **status**: accepted --> pending

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/171/#1359) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-12-04 09:37:46.381000:

fixed in [5b7ae8103876c8efc0d235bb14ed462326e7ea7c] this needs testing

**Comment by joey-coleman** [comment]
> [Comment](https://sourceforge.net/p/compassresearch/bugs/171/#c258) by [lausdahl](https://sourceforge.net/u/lausdahl/profile/), 2013-12-10 20:00:58.989000:
- **status**: pending --> closed
